contracts/UFragmentsPolicy.sol

```diff
@@ -177,8 +191,11 @@ contract UFragmentsPolicy is Ownable {
     {
         Ownable.initialize(owner);
 
-        deviationThreshold = (5 * TARGET_RATE) / 100;  // 5% of target
-        minimumVolume = 1;
+        targetRate = INITIAL_CPI.mul(10 ** RATE_DECIMALS).div(TARGET_CPI);
```

Comment:
storing INITIAL_CPI and computing targetRate in initialize isn't needed.

Reply:
s/RATE_DECIMALS/CPI_DECIMALS

Reply:
👍